src/ESPiLight.cpp

```diff
@@ -54,7 +54,7 @@ void ESPiLight::initReceiver(byte inputPin) {
   enableReceiver();
 
   if (interrupt >= 0) {
-    attachInterrupt(interrupt, interruptHandler, CHANGE);
+    attachInterrupt(static_cast<uint8_t>(interrupt), interruptHandler, CHANGE);
```

Comment:
I am not yet sure why we need all this explicit type casting. Did we get any warning? What can go wrong here, if we use automatic casting?

Reply:
I used clion and their linter complained that there "might" be an issue. As I like to have a "green" file to see if I introduced any problems I cleaned all the "issues" by giving explicit casts.

Also it shows that there happens a type-conversion. So its easier to spot bugs as you can follow the data path and see where which type is actually used.

Reply:
OK, this could be a argument. Hopefully, there will be no other person with yet another linter.  But `(uint8_t)` would be enough.